test/Confluent.Kafka.StrongName.UnitTests/Confluent.Kafka.StrongName.UnitTests.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <AssemblyName>Confluent.Kafka.StrongName.UnitTests</AssemblyName>
-    <TargetFrameworks>netcoreapp1.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.1;</TargetFrameworks>
```

Comment:
Still need to test against netframework - the lib is netstandard, but it should work on all implementation of netstandard, mainly netcoreapp (perhaps test netcoreapp2 also) and net461